runhouse/resources/hardware/sky/command_runner.py

```diff
@@ -648,3 +648,193 @@ def rsync(
                     log_path=log_path,
                     stream_logs=stream_logs,
                     max_retry=max_retry)
+
+
+class KubernetesCommandRunner(CommandRunner):
+    """Runner for Kubernetes commands."""
+
+    def __init__(
+        self,
+        node: Tuple[str, str],
+        **kwargs,
+    ):
+        """Initialize KubernetesCommandRunner.
+
+        Example Usage:
+            runner = KubernetesCommandRunner((namespace, pod_name))
+            runner.run('ls -l')
+            runner.rsync(source, target, up=True)
+
+        Args:
+            node: The namespace and pod_name of the remote machine.
+        """
+        del kwargs
```

Comment:
I don't get this... but ok

Reply:
this file is just source code from skypilot, that we subclass and edit for the runhouse versions. I guess the only core components for the k8s command runner are the namespace and pod name